Remove USB from EX3700/3800

maurerle opened the pull request:
As this device does not have a lot of Flash left which softbricked one device for me, I would suggest to remove USB as this is not available for those devices anyways.

GoliathLabs reviewed (changes requested):
Can you also run the script and change the site.mk file accordingly?

maurerle committed:
Remove USB from EX3700/3800

As this device does not have a lot of Flash left which softbricked one device for me, I would suggest to remove USB as this is not available for those devices anyways.

maurerle commented:
Sorry, I forgot about it - should be ready to merge

GoliathLabs commented:
@maurerle thanks for the work. Will wait until the CI checks completed successfully.
